### PR TITLE
Feature #18 - Define and execute refactoring plan for BibEntry::getSourceField

### LINKED ISSUE #18: Define and execute refactoring plan for BibEntry::getSourceField

### PR DESCRIPTION
**Pull request by emiros98**
closes #18

### CODE REVIEW

**Comment by pat672** [comment]
Looks like this branch needs to resolve a conflict in the readme file @emiros98

**Comment by emiros98** [comment]
Resolved! @pat672

File: dd2480/README.md [hunk 1]
```diff
@@ -281,7 +281,20 @@ There are many opportunities to split up the code into smaller functions. We wou
...
+
+which can be refactored into:
+
+`if (sourceEntry == x) {evaluateTargetEntry(targetEntry)}`
+
 ## Task 3 - Refactoring Implementation
 
 ### RisImporter::importDatabase
 By applying the refactoring plan, we have managed to reduce CCN from 110 down to 71 which constitutes a 35% reduction
+
```
Comment (pat672) [inline dd2480/README.md:345]: Could we also document the outcome of the refactoring for FieldNameLabel? Write down what CCN we started with and the reduction after refactoring

**Review by pat672 — CHANGES_REQUESTED** [review]
LGTM, just one small comment!

**Comment by emiros98** [comment]
All should be good now

**Review by pat672 — APPROVED** [review]
LGTM

**Pull request merged**